os/include/tinyara/binary_manager.h

```diff
@@ -142,17 +142,15 @@ struct binary_header_s {
 	uint32_t bin_ver;
 	uint32_t bin_ramsize;
 	uint32_t bin_stacksize;
-	float kernel_ver;
-	uint32_t jump_addr;
+	uint16_t kernel_ver;
```

Comment:
If we rearrange the member variable, we can save the memory, Because of padding.
Please check it.